p5-travel-app/src/client/js/form.js

```diff
@@ -0,0 +1,28 @@
+const getCity = () => {
+  const city = document.getElementById("city").value;
+  city = city.toLowerCase();
+  city = city[0].toUpperCase() + city.slice(1);
+  console.log(city);
```

Comment:
Remove

Reply:
done in [77541ac](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/9/commits/77541acca892e61c7f81d34c2e31e59fbb1a7011)